roles/hostname/templates/config.j2

```diff
@@ -0,0 +1 @@
+{{ inventory_hostname_short }}{{ '\n' }}
```

Comment:
You can drop the `{{ '\n' }}` part, the newline is implicit in the template. This was only needed for the copy/content variant.

Reply:
You are right i oversee that, thank you.